.github/workflows/unittests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install -U coverage pytest flake8 pydocstyle
+        python -m pip install -U coverage pytest flake8 pydocstyle check-manifest pytest-cov
```

Comment:
Check this because git actions raise an error.

Reply:
Done